repository/repo.go

```diff
@@ -30,24 +30,33 @@ type RepoCommon interface {
 	// GetRemotes returns the configured remotes repositories.
 	GetRemotes() (map[string]string, error)
 
-	// StoreConfig store a single key/value pair in the config of the repo
+	RepoConfig
+}
+
+type RepoConfig interface {
+	// StoreConfig store a single key/value pair in the config of the repos
 	StoreConfig(key string, value string) error
+	StoreGlobalConfig(key string, value string) error
```

Comment:
It's very much readable for a human reader, but the godoc police will complain if there is no doc for those exported functions.

Reply:
Just commented every public function/interface in https://github.com/MichaelMure/git-bug/pull/219/commits/d0ea12f8601671233866754d100fe616c08b7e6a